src/app/static/src/app/components/surveyAndProgram/SurveyAndProgram.vue

```diff
@@ -8,7 +8,7 @@
                        v-translate="'map'"
                        v-on:click="selectTab(0)"></a>
                 </li>
-                <li class="nav-item">
+                <li class="nav-item" v-if="anc.available || programme.available">
```

Comment:
Thought I'd added a comment here but can't see it now: was just going to suggest that you could do the following as a way to avoid doing the "anc or art" check in two places? But completely up to you.
```suggestion
                <li class="nav-item" v-if="availableDatasetIds.length">
```